kinto/core/storage/postgresql/client.py

```diff
@@ -75,29 +74,28 @@ def create_from_config(config, prefix='', with_transaction=True):
     from sqlalchemy.orm import sessionmaker, scoped_session
 
     settings = {**config.get_settings()}
+
     # Custom Kinto settings, unsupported by SQLAlchemy.
-    settings.pop(prefix + 'backend', None)
-    settings.pop(prefix + 'max_fetch_size', None)
-    settings.pop(prefix + 'max_size_bytes', None)
-    settings.pop(prefix + 'prefix', None)
-    settings.pop(prefix + 'strict_json', None)
-    settings.pop(prefix + 'hosts', None)
-    transaction_per_request = with_transaction and settings.pop('transaction_per_request', False)
-
-    url = settings[prefix + 'url']
+    black_list = create_black_list(prefix)
```

Comment:
Instead of having a function, you could just define a constant `EXCLUDE_SETTINGS = ('prefix', 'hosts', ...)` somewhere on top.